nsqadmin/static/js/views/channel.js

```diff
@@ -0,0 +1,47 @@
+var $ = require('jquery');
+
+window.jQuery = $;
+var bootstrap = require('bootstrap'); //eslint-disable-line no-unused-vars
+var bootbox = require('bootbox');
+
+var Pubsub = require('../lib/pubsub');
+var AppState = require('../app_state');
+
+var BaseView = require('./base');
+
+var ChannelView = BaseView.extend({
+    className: 'channel container-fluid',
+
+    template: require('./spinner.hbs'),
+
+    events: {
+        'click .channel-actions button': 'channelAction'
+    },
+
+    initialize: function() {
+        BaseView.prototype.initialize.apply(this, arguments);
+        this.listenTo(AppState, 'change:graph_interval', this.render);
+        this.model.fetch().done(function() {
```

Comment:
From offline convo: might not be worth making this change, as the payoff is mostly in terms of future-proofing the code. But it sounds like we want to do a React rewrite anyway. So let's not worry about this right now.